### PR TITLE
Feature/avatar support

### PR DESCRIPTION
**Pull request by kirillta**

### CODE REVIEW

File: TipCatDotNet.Api/Controllers/AccountAvatarManagementController.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    /// </summary>
+    /// <param name="accountId">Target account ID</param>
+    /// <param name="file">Avatar as a form file</param>
+    /// <returns></returns>
+    [HttpPost]
+    [HttpPut]
+    [RequestSizeLimit(5 * 1024 * 1024)]
+    [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+    public async Task<IActionResult> AddOrUpdateMemberAvatar([FromRoute] int accountId, [FromForm] IFormFile? file)
```
Comment (DaniyarAskarov) [inline TipCatDotNet.Api/Controllers/AccountAvatarManagementController.cs:36]: Should it be AddOrUpdateAccountAvatar?

File: TipCatDotNet.Api/Controllers/FacilityAvatarManagementController.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    /// <param name="accountId">Target account ID</param>
+    /// <param name="facilityId">Target facility ID</param>
+    /// <param name="file">Avatar as a form file</param>
+    /// <returns></returns>
+    [HttpPost]
+    [HttpPut]
+    [RequestSizeLimit(5 * 1024 * 1024)]
+    [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+    public async Task<IActionResult> AddOrUpdateMemberAvatar([FromRoute] int accountId, [FromRoute] int facilityId, [FromForm] IFormFile? file)
```
Comment (DaniyarAskarov) [inline TipCatDotNet.Api/Controllers/FacilityAvatarManagementController.cs:37]: The same as in accountAvatar adding

**Review by DaniyarAskarov — APPROVED** [review]

**Comment by sergeyf-QA** [comment]
@kirillta 

1) I can upload an avatar w/o an aspect ratio close to 1:1, e.g. 960x518
960x518

2) when I try to download a 5mb or more picture it returns me
{
  "status": 400,
  "detail": "Can't read the file. Probably, the Content-Description header isn't set to 'multipart/form-data'."
}
Perhaps, it would be great to have something meaningful in this response "detail"?

**Comment by sergeyf-QA** [comment]
**Summary:**
1) 
DELETE /api​/accounts​/{accountId}​/avatar
DELETE ​/api​/accounts​/{accountId}​/facilities​/{facilityId}​/avatar
DELETE ​/api​/accounts​/{accountId}​/members​/{memberId}​/avatar
remove avatars well

2)
POST ​/api​/accounts​/{accountId}​/avatar
PUT /api/accounts/{accountId}/avatar
POST ​/api​/accounts​/{accountId}​/facilities​/{facilityId}​/avatar
PUT ​/api​/accounts​/{accountId}​/facilities​/{facilityId}​/avatar
POST /api/accounts/{accountId}/members/{memberId}/avatar
PUT /api/accounts/{accountId}/members/{memberId}/avatar
add/update avatars according to rules:
- size < 5mb
- aspect ratio 1:1
- dimensions from 188x188
- jpg, jpeg, png

**OK on DEV.**

**Pull request merged**